src/cliAction/startStopDelete/handler_cmdSwitch.go

```diff
@@ -0,0 +1,55 @@
+package startStopDelete
+
+import (
+	"context"
+
+	"github.com/zerops-io/zcli/src/constants"
+	"github.com/zerops-io/zcli/src/i18n"
+)
+
+type Method func(ctx context.Context, h *Handler, projectId string, serviceId string) (string, error)
+
+type CmdType struct {
+	Start   string
+	Finish  string
+	Execute Method
+}
+
+func (h *Handler) getCmdProps(parentCmd constants.ParentCmd, childCmd constants.ChildCmd) (string, string, Method) {
+	switcher := make([][]CmdType, 2)
```

Comment:
I would populate the switcher once on start for example in the `New` method, because it's constant and `getCmdProps` should no populate that.

Reply:
maybe in `register` function or `method`